### PR TITLE
feat: lazyRender returns this

### LINKED ISSUE #3551: lazyRender should return this to preserve chaining

### PR DESCRIPTION
**Pull request by jbrundage**
Fixes #3551

Signed-off-by: Jaman Brundage <jbrundage372@gmail.com>

## Checklist:
- [x] The commit message is properly formatted and free of typos.
  - [x] The commit message title makes sense in a changelog, by itself.
  - [x] The commit message includes a "fixes" reference if appropriate.
  - [x] The commit is signed.
- [x] The change has been fully tested:
  - [ ] I have viewed all related gallery items
  - [ ] I have viewed all related dermatology items
- [x] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised new issues to address them separately

### CODE REVIEW

File: packages/common/src/Widget.ts [hunk 1]
```diff
@@ -652,9 +652,12 @@ export abstract class Widget extends PropertyExt {
         });
     }
 
-    lazyRender = debounce(function (debouncedCallback?: (w: Widget) => void) {
-        this.render(debouncedCallback);
-    }, 100);
+    lazyRender(): this {
+        debounce(function (debouncedCallback?: (w: Widget) => void) {
+            this.render(debouncedCallback);
+        }, 100);
```
Comment (GordonSmith) [inline packages/common/src/Widget.ts:658]: This looks like if lazyRender is called multiple times within 100ms that render will also get called multiple times?

Comment (jbrundage) [reply packages/common/src/Widget.ts:658]: Hmmmm, good point. How can this be avoided? Cache the callback function and compare against subsequent calls?

Comment (jbrundage) [reply packages/common/src/Widget.ts:658]: ...or maybe return `this` from within `debounce`? But `debounce` returns a `debounced` function. I'll give this issue another look from the perspective of rapidly calling it.

Comment (GordonSmith) [reply packages/common/src/Widget.ts:658]: rename to `private _lazyRender` and then have a new lazyRender call it and return this.

Comment (GordonSmith) [reply packages/common/src/Widget.ts:658]: @jbrundage bump

File: packages/common/src/Widget.ts [hunk 2]
```diff
@@ -652,10 +652,15 @@ export abstract class Widget extends PropertyExt {
...
 
-    lazyRender = debounce(function (debouncedCallback?: (w: Widget) => void) {
+    private _lazyRender = debounce(function (debouncedCallback?: (w: Widget) => void) {
         this.render(debouncedCallback);
     }, 100);
 
+    lazyRender(): this {
+        this._lazyRender();
+        return this;
+    }
```
Comment (jbrundage) [inline packages/common/src/Widget.ts:662]: @GordonSmith is this what you had in mind?

Comment (GordonSmith) [reply packages/common/src/Widget.ts:662]: Yup.

**Pull request merged**